pkg/deployer/lib/crdresources/deployer.landscaper.gardener.cloud_owners.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: owners.deployer.landscaper.gardener.cloud
+spec:
+  group: deployer.landscaper.gardener.cloud
+  names:
+    kind: Owner
+    plural: owners
+    shortNames:
+    - ow
+    singular: owner
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.type
+      name: Type
+      type: string
+    - jsonPath: .spec.deployerId
+      name: Deployer
+      type: string
+    - jsonPath: .status.accepted
+      name: Accepted
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: Owner are resources that can hold any kind of json or yaml data.
+        properties:
+          spec:
+            description: Spec contains configuration for a deployer ownership.
+            properties:
+              deployerId:
+                description: DeployerId describes the unique identity of a deployer.
+                type: string
+              targets:
+                description: Targets is a list of targets the referenced deployer
+                  is responsible for.
+                items:
+                  description: ObjectReference is the reference to a kubernetes object.
+                  properties:
+                    name:
+                      description: Name is the name of the kubernetes object.
+                      type: string
+                    namespace:
+                      description: Namespace is the namespace of kubernetes object.
+                      type: string
+                  required:
+                  - name
+                  type: object
+                type: array
+              type:
```

Comment:
Just a general question. If the deployer ID defines a deployer, why do we need the type field?

Reply:
you're right that the type field is not necessary but its more for readability.
So that human operators can better correlate.